### PR TITLE
Import Linux Kernel typos: letter A

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -849,19 +936,23 @@ applys->applies
...
+appopriate->appropriate
 apporpiate->appropriate
 apppear->appear
 apppend->append
 appplet->applet
 appplication->application
 appplications->applications
+apppriate->appropriate
 apprearance->appearance
+apprended->appended
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can also be `apprehended`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -205,6 +222,7 @@ achor->anchor
 achored->anchored
 achoring->anchoring
 achors->anchors
+aci->acpi
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: https://github.com/codespell-project/codespell/pull/567#pullrequestreview-137237332
(should we capitalize this?)

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -181,8 +195,11 @@ aceptable->acceptable
 acerage->acreage
 acess->access
 acessable->accessible
+acesses->accesses
+acessible->accessible
 acessing->accessing
 acessor->accessor
+acessors->accessors
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please also add `accessor` per https://github.com/codespell-project/codespell/pull/567#pullrequestreview-137237297

**Comment by luzpaz** [comment]
only after the abovementioned revisions and rebase this commit LGTM

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: add

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: added

**Comment by Gelma** [comment]
I'm going to close this PR and open new one.

**Pull request closed without merging**